gqcpy/test_bindings.py

```diff
@@ -0,0 +1,2 @@
+import gqcpy
```

Comment:
Could you put this test in the `gqcpy/tests/` folder?

Reply:
Addressed in latest commit.